### ISSUE TITLE
Mpris module not showing non-square album art

### ISSUE THREAD

**Issue by petruckov** [Aylur/ags#37]
Hi, I just recently switched to ags and it's been amazing so far. Just found a small issue with the Mpris module not showing album art when the album art is rectangle and wanted to let you know

**Comment by petruckov** [comment]
wait actually it's not an issue with the shape, on some other mp3 files it still shows, not sure why. I've tried both vlc and elisa with the same result

**Comment by Aylur** [comment]
I have just tested both and they seem to work.
Are you sure the mp3 files contain the metadata for the artwork?

**Comment by petruckov** [comment]
Yea, I even tried to rewrite the metadata with puddletag, same result. I tried with a different image as well but it also didn't show up. Is there a way to see logs for ags?

**Comment by Aylur** [comment]
only logs are the stdout you see when running it in a terminal

**Comment by petruckov** [comment]
I just ran ags in terminal and opened the mp3 file:

(ags:49900): Gjs-CRITICAL **: 09:42:32.900: JS ERROR: GLib.SpawnError: Failed to execute child process “convert” (No such file or directory)
execAsync@resource:///com/github/Aylur/ags/js/utils.js:161:33
Widget.widgets["mpris/blurred-cover-art"]/<.connections<@/home/user/.config/ags/modules/mpris.js:73:18
connect/<@resource:///com/github/Aylur/ags/js/utils.js:83:31
timeout/<@resource:///com/github/Aylur/ags/js/utils.js:99:9
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34


(ags:49900): Gjs-CRITICAL **: 09:42:32.987: JS ERROR: GLib.SpawnError: Failed to execute child process “convert” (No such file or directory)
execAsync@resource:///com/github/Aylur/ags/js/utils.js:161:33
Widget.widgets["mpris/blurred-cover-art"]/<.connections<@/home/user/.config/ags/modules/mpris.js:73:18
connect/bind<@resource:///com/github/Aylur/ags/js/utils.js:81:66
_addPlayer/<@resource:///com/github/Aylur/ags/js/service/mpris.js:194:18
_updateState@resource:///com/github/Aylur/ags/js/service/mpris.js:98:14
timeout/<@resource:///com/github/Aylur/ags/js/utils.js:99:9
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34

**Comment by Aylur** [comment]
this just means you are missing the `imagemagick` package

**Comment by petruckov** [comment]
thanks for pointing that out lol, i can see the background blur now (on other mp3s). But the cover image still doesn't show for that one file

**Comment by Aylur** [comment]
thats weird if the background works, the other one should, since that one gets converted
try running `ags clear-cache`

**Comment by petruckov** [comment]
ah sorry I was unclear earlier, I mean for that particular file both the background and the cover doesn't work. I have tried to run `ags clear-cache` too

**Comment by petruckov** [comment]
I think I figured out what caused the problem. When I checked the cache folder, I noticed that the cover cache file names only contains non-Japanese characters. And that particular file I mentioned has both the artist name and the song name in Japanese characters. On my other mp3 files the songs all have at least the artist name or the song name in English, and their cover cache file names only have the non-Japanese part
